### PR TITLE
How to guide on Ballerina update tool

### PR DESCRIPTION
**Pull request by sameerajayasoma**
This feature will be released with Ballerina 1.1.0 version

## Check List

- [x] **Page Addition**
  - [x] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: learn/how-to-keep-ballerina-up-to-date.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
+---
+layout: ballerina-inner-page
+title: How to Keep Ballerina up to date
+permalink: /learn/how-to-keep-ballerina-up-to-date/
+---
+
+# How to Keep Ballerina up to date
+This guide explains how to maintain your Ballerina installation up to date with the latest patch and minor releases. If you haven’t installed Ballerina yet, visit [Getting Started](https://ballerina.io/learn/getting-started/) guide.
```
Comment (anupama-pathirage) [inline learn/how-to-keep-ballerina-up-to-date.md]: Getting started has changed to "Installing Ballerina" with the PR [1] . Should we udpate the link with this?

Comment (keizer619) [reply learn/how-to-keep-ballerina-up-to-date.md]:
```suggestion
This guide explains how to maintain your Ballerina installation up to date with the latest patch and minor releases. If you haven’t installed Ballerina yet, visit [installing Ballerina](https://ballerina.io/learn/getting-started/). 
```

File: learn/how-to-keep-ballerina-up-to-date.md [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+       pull       Fetch a distribution and set it as the active version
+       use        Set a distribution as the active distribution
+       list       List locally and remotely available distributions
+       remove     Remove distributions in your local environment
+
+
+Use 'ballerina help dist <command>' for more information on a specific command.
+```
+
+Most of these subcommands are self-explanatory. Therefore following sections briefly introduce them.
```
Comment (MaryamZi) [inline learn/how-to-keep-ballerina-up-to-date.md]:
```suggestion
Most of these subcommands are self-explanatory. Therefore, the following sections introduce them briefly.
```

File: learn/how-to-keep-ballerina-up-to-date.md [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+
+### Update to latest patch version
+The `ballerina dist update` command updates your active distribution to the latest patch version.
+
+E.g., If the active distribution in your environment is “jballerina-1.1.0” and there exists patch version “jballerina-1.1.4” in our servers, this command will fetch and set it as the active.
```
Comment (MaryamZi) [inline learn/how-to-keep-ballerina-up-to-date.md]:
```suggestion
E.g., If the active distribution in your environment is “jballerina-1.1.0” and there exists patch version “jballerina-1.1.4” in our servers, this command will fetch and set it as the active distribution.
```

**Pull request merged**